pkg/virtualKubelet/reflection/namespacemap/fake/namespaceStartStopper.go

```diff
@@ -0,0 +1,44 @@
+// Copyright 2019-2021 The Liqo Authors
```

Comment:
Filenames should be short, or snake_case. Additionally, would move this to the `fake` directory of the manager (since it implements that interface).